src/types/index.tsx

```diff
@@ -30,51 +30,6 @@ export type Resource = {
   date?: string;
 };
 
-export interface PersonaProps {
-  personaIndex: number;
-  details: Persona;
-}
```

Comment:
moved all these types to local component. should that component no longer relevant, can delete the folder entirely